components/admin_console/cloud_migrate/widgets/cloud_table.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import './cloud_table.scss';
+
+type CloudTableProps = {
```

Comment:
Is there anything that makes this component cloud specific? Seems to be a generic table that could be reused later?

Reply:
nothing, I just though that potentially there could be other styles of tables. So I picked that name to make it easier to understand this table has the cloud style.